src/clientiface.cpp

```diff
@@ -199,7 +199,10 @@ void RemoteClient::GetNextBlocks (
 	// Distrust client-sent FOV and get server-set player object property
 	// zoom FOV (degrees) as a check to avoid hacked clients using FOV to load
 	// distant world.
-	float prop_zoom_fov = sao->getZoomFOV() * core::DEGTORAD;
+	// (zoom is disabled by value 0)
+	float prop_zoom_fov = sao->getZoomFOV() < 0.001f ?
```

Comment:
Any reason why using `0.001f` and not other value?

Reply:
See https://github.com/minetest/minetest/pull/7234#pullrequestreview-112214472